lib/src/visitor/async_evaluate.dart

```diff
@@ -512,7 +539,62 @@ final class _EvaluateVisitor
             configuration: configuration,
             namesInErrors: true);
         _assertConfigurationIsEmpty(configuration, nameInError: true);
-      }, url: "sass:meta")
+      }, url: "sass:meta"),
+      BuiltInCallable.mixin("apply", r"$mixin, $args...", (arguments) async {
+        var mixin = arguments[0];
+        var args = arguments[1] as SassArgumentList;
+
+        var callableNode = _callableNode!;
+        var invocation = ArgumentInvocation(
+          [],
+          {},
+          callableNode.span,
+          rest: ValueExpression(args, callableNode.span),
+        );
+
+        var callable = mixin.assertMixin("mixin").callable;
+        var content = _environment.content;
+
+        switch (callable) {
+          case AsyncBuiltInCallable(acceptsContent: false) when content != null:
+            throw _exception(
+                "Mixin doesn't accept a content block.", callableNode.span);
+
+          case AsyncBuiltInCallable():
+            await _environment.withContent(content, () async {
+              await _runBuiltInCallable(invocation, callable, callableNode);
+            });
+
+          case UserDefinedCallable<AsyncEnvironment>(
+                declaration: MixinRule(hasContent: false)
+              )
+              when content != null:
+            throw MultiSpanSassRuntimeException(
+                "Mixin doesn't accept a content block.",
+                callableNode.span,
+                "invocation",
+                {callable.declaration.arguments.spanWithName: "declaration"},
+                _stackTrace(callableNode.span));
+
+          case UserDefinedCallable<AsyncEnvironment>():
+            _runUserDefinedCallable(invocation, callable, callableNode,
+                () async {
+              await _environment.withContent(content, () async {
```

Comment:
Do you need to call this? Won't the current content just remain the same otherwise?